src/app/games/rectanglz/RectanglzTutorial.ts

```diff
@@ -0,0 +1,114 @@
+            RectanglzMove.from(new Coord(7, 0), new Coord(5, 2)).get(),
+            (move: RectanglzMove, _: RectanglzState) => {
+                if (move.isDuplication()) {
+                    return MGPValidation.failure($localize`This was a single step, try a jump now.`);
+                } else {
+                    return MGPValidation.SUCCESS;
+                }
+            },
+            TutorialStepMessage.CONGRATULATIONS(),
+        ).withPreviousMove(RectanglzMove.from(new Coord(0, 0), new Coord(1, 1)).get()), // TODO: check that we SEE the highlight
+        TutorialStep.fromPredicate(
+            $localize`Captures`,
+            $localize`When one of your pieces arrives on a square neighbor to opponent's pieces, those pieces switch color.<br/>You're playing Dark, do such a move!`,
+            new RectanglzState([
+                [O, _, _, _, _, _, _, X],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, O, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, X, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, O],
+            ], 3),
+            RectanglzMove.from(new Coord(2, 5), new Coord(3, 4)).get(),
+            (_move: RectanglzMove, _state: RectanglzState, resultingState: RectanglzState) => {
+                if (resultingState.getPieceAtXY(3, 3) === Player.ONE) {
+                    return MGPValidation.SUCCESS;
+                } else {
+                    return MGPValidation.failure(TutorialStepMessage.YOU_DID_NOT_CAPTURE_ANY_PIECE());
+                }
+            },
+            TutorialStepMessage.CONGRATULATIONS(),
+        ).withPreviousMove(RectanglzMove.from(new Coord(1, 1), new Coord(3, 3)).get()),
+        TutorialStep.fromPredicate(
+            $localize`Captures`,
+            $localize`When one of your pieces arrives on a square neighbor to opponent's pieces, those pieces switch color.<br/>You're playing Dark, do such a move!`,
+            new RectanglzState([
+                [X, X, X, X, X, X, X, X],
+                [O, O, O, O, O, O, O, O],
+                [O, O, O, O, O, O, O, O],
+                [O, O, O, O, O, O, O, O],
+                [X, X, X, _, X, X, X, X],
+                [X, X, X, X, X, X, X, X],
+                [X, X, X, X, X, X, X, X],
+                [O, O, O, O, O, O, O, O],
+            ], 3),
+            RectanglzMove.from(new Coord(2, 5), new Coord(3, 4)).get(),
+            (move: RectanglzMove, _state: RectanglzState, _resultingState: RectanglzState) => {
+                if (move.isDuplication()) {
+                    return MGPValidation.SUCCESS;
+                } else {
+                    return MGPValidation.failure($localize`Bad choice, by making this move you allowed the opponent to steal you 5 pieces, win 5 pieces, and duplicate one piece, hence taking 11 points of advance on you and winning.<br/>Try again!`);
```

Comment:
to steal 5 of your pieces*
attends, il a VOLÉ 5, GAGNÉ 5, et dupliqué 1 ?  comment il peut voler et gagner en même temps ? Quelle sorcellerie ! Puis sur un plateau où je teste ça, bah il prends certes 11 points par rapport à avant mon tour, mais il me vole 6 pièces, pas 5 (car il me vole une que je lui ai volé)

Reply:
Bah, si je retourne 5 de tes pièces, tu perds 5 point, et moi j'en gagne 5, et le 11ème point gagné c'est via la duplication :p
![image](https://github.com/EveryBoard/EveryBoard/assets/41200522/3e11ebf3-54be-42bd-a9fa-2f8180e85448)
T'as tu eu ce move là ? Pour sûr moi j'ai pas fait le même, j'ai sauté de l'intérieur des terres vers la frontière, ça donne différent !

Reply:
FOERTE, IL Y A PIRE
![image](https://github.com/EveryBoard/EveryBoard/assets/41200522/cf00c560-99d8-42dd-b24d-553f785d1520)

Reply:
voilà j'ai simpilifié le message

Reply:
mieux ! 
Pour répondre si tu me fait perdre 5 points pour en gagner 5, tu m'as volé 5 points. Mais NON tu as pas volé 5 points ET gagné 5 points (t'as gagné 5 points que tu m'as volé, mais c'est les mêmes).
TPR

Reply:
C'est vrai, tu vole 5 point, switchant le diff de 10 point en ta faveur mais c'pas "voler 5 et gagner 5", c'est des nomenclatures un peu meh.

TPR !